corehq/motech/fhir/models.py

```diff
@@ -298,3 +303,29 @@ def get_resource_type_or_none(case, fhir_version) -> Optional[FHIRResourceType]:
         )
     except FHIRResourceType.DoesNotExist:
         return None
+
+
+class FHIRImporter(models.Model):
+    domain = models.CharField(max_length=127, db_index=True)
+    connection_settings = models.ForeignKey(
+        ConnectionSettings,
+        on_delete=models.PROTECT,
+    )
+    fhir_version = models.CharField(
+        max_length=12,
+        choices=FHIR_VERSIONS,
+        default=FHIR_VERSION_4_0_1,
+    )
+    frequency = models.CharField(
+        max_length=12,
+        choices=IMPORT_FREQUENCY_CHOICES,
+        default=IMPORT_FREQUENCY_DAILY,
+    )
+    # ID of user or location that will own imported cases
```

Comment:
You might also want a `owner_type` field if it's ever necessary to lookup what the owner is - it's awkward to have to do multiple DB calls, and this would be trickier to add down the road and back-populate.